com.jetbrains.IntelliJ-IDEA-Community.json

```diff
@@ -1,22 +1,48 @@
 {
   "app-id": "com.jetbrains.IntelliJ-IDEA-Community",
   "command": "idea",
-  "runtime": "org.freedesktop.Sdk",
-  "runtime-version": "18.08",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
   "separate-locales": false,
-  "sdk": "org.freedesktop.Sdk",
-  "finish-args": [
-    "--device=all",
-    "--device=dri",
-    "--env=IDEA_JDK=/app/idea-IC/jre64/",
-    "--env=IDEA_VM_OPTIONS=/app/idea-IC/bin/idea64.vmoptions",
-    "--filesystem=host",
-    "--share=ipc",
-    "--share=network",
-    "--socket=x11",
-    "--talk-name=org.freedesktop.Notifications",
-    "--talk-name=org.freedesktop.secrets"
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.openjdk9",
+    "org.freedesktop.Sdk.Extension.openjdk10",
+    "org.freedesktop.Sdk.Extension.openjdk11"
   ],
+  "finish-args" : [
+        "--require-version=0.10.0",
+        "--allow=devel",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--filesystem=host",
+        "--share=network",
+        "--talk-name=org.gtk.vfs.*",
+        "--system-talk-name=org.freedesktop.PolicyKit1",
+        "--system-talk-name=org.gnome.Sysprof2",
+        "--talk-name=org.gnome.CodeAssist.v1.*",
+        "--filesystem=xdg-run/dconf",
```

Comment:
IntelliJ IDEA does not store anything via `GSettings` as far as I know. What did you see that needs `DConf` access?